src/ontology/uberon-edit.obo

```diff
@@ -224246,6 +224246,119 @@ intersection_of: part_of UBERON:0001987 ! placenta
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-06-27T09:30:52Z xsd:dateTime
 
+[Term]
+id: UBERON:8480009
+name: perianal space
+def: "The virtual spaces circumscribed by the mucosa lining of the anal canal and the muscular structures of the sphincter complex." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:0000464 ! anatomical space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0001353 ! anal region
+property_value: dcterms-date 2022-07-05T10:24:30Z xsd:dateTime
+
+[Term]
+id: UBERON:8480010
+name: subcutaneous perianal space
+def: "A space located below the dentate line. Its borders are the skin of the anal verge below and the internal sphincter above." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:36:29Z xsd:dateTime
+
+[Term]
+id: UBERON:8480011
+name: submucous perianal space
+def: "A space located above the dentate line. It is delimited by the internal sphincter and by the anoderm." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:41:02Z xsd:dateTime
+
+[Term]
+id: UBERON:8480012
+name: intersphincteric perianal space
+def: "A space located between the internal and external anal sphincter." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+synonym: "Eisenhammer’s space" EXACT [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:43:20Z xsd:dateTime
+
+[Term]
+id: UBERON:8480013
+name: ischiorectal perianal space
+def: "A large space outside the anal canal. It is delimited externally by the perianal skin and the gluteus, internally by the external sphincter and above by the levator ani muscle and internal obturator muscle. The two contralateral ischiorectal spaces communicate posteriorly around anorectal region via deep postanal space, located anteriorly to the ano-coccygeal ligament." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
```

Comment:
"A perianal space outside the anal canal that is delimited externally by..."

Reply:
@paulacalipho, is this the same as the ischioanal (aka ischiorectal) fossa?
https://www.ncbi.nlm.nih.gov/books/NBK535387/

The above link mentions some of the other spaces included in this ticket in case you'd like to review those as well.

Reply:
@bvarner-ebi seems to me that in this book the spaces are not exactly classified the same. 
The ischioanal fossa or ischiorectal fossa is divided into the perianal space and the ischiorectal space. 
Then authors describe that the different anorectal spaces are: 
1- the perianal space, 
2- the ischiorectal space (or ischioanal space), 
3- the intersphincteric space, 
4- supra levator space (or pelvic-rectal space), 
5- the deep post-anal space. 
Not clear to me if I should add as synonyms the following terms and add as ref the book:
ischiorectal perianal space -> ischiorectal space and ischioanal space
intersphincteric perianal space -> intersphincteric space

Reply:
@paulacalipho, the label you have for this term is 'ischiorectal perianal space', and after a quick search, I did not find this term used outside of your source. The boundaries in the definition are a bit vague and the definition also mentions the 'ischiorectal spaces'-- are these the same as 'ischiorectal perianal spaces'?

The NIH link provides terms that seem to be more universally used (again, based on a quick online search). 
Does this new term represent...
A:  'ischioanal fossa' (synonym: ischiorectal fossa)
B:  the 'perianal space of the ischioanal fossa'
C:  the 'ischiorectal space of the ischioanal fossa'
D:  None of the above/something else

Reply:
@bvarner-ebi 'ischiorectal perianal space' refers to C (the 'ischiorectal space of the ischioanal fossa'). I agree with you that NIH link provides terms that seem to be more universally used. I also found this book that describe these spaces. I will change the classification according to NIH book (https://books.google.ch/books?id=TaHWCwAAQBAJ&pg=PA11&lpg=PA11&dq=%22Ischioanal+space%22&source=bl&ots=zhLTOwSZJB&sig=ACfU3U0f5u97EvxkIsQJl9Iy5jQTHhi61w&hl=en&sa=X&ved=2ahUKEwiApaLF5bb5AhUhgv0HHSj4DmgQ6AF6BAgfEAM#v=onepage&q=%22Ischioanal%20space%22&f=false)